components/org.wso2.carbon.identity.tenant.resource.manager/pom.xml

```diff
@@ -0,0 +1,126 @@
+            <groupId>org.jacoco</groupId>
+            <artifactId>org.jacoco.agent</artifactId>
+            <classifier>runtime</classifier>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-module-testng</artifactId>
+            <version>${powermock.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-api-mockito</artifactId>
+            <version>${powermock.version}</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>3.2.0</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Private-Package>
+                            org.wso2.carbon.identity.tenant.resource.manager.internal,
+                        </Private-Package>
+                        <Export-Package>
+                            !org.wso2.carbon.identity.tenant.resource.manager.internal,
+                            org.wso2.carbon.identity.tenant.resource.manager.*,
+                        </Export-Package>
+                        <Import-Package>
+                            org.apache.commons.logging.*; version="${commons-logging.osgi.version.range}",
+                            org.apache.commons.collections; version="${commons-collections.wso2.osgi.version.range}",
+                            org.osgi.framework; version="${osgi.framework.imp.pkg.version.range}",
+                            org.osgi.service.component; version="${osgi.service.component.imp.pkg.version.range}",
+                            org.wso2.carbon.context; version="${carbon.kernel.package.import.version.range}",
+                            org.wso2.carbon.core.util; version="${carbon.kernel.package.import.version.range}",
+                            org.wso2.carbon.databridge.commons; version="${carbon.analytics.common.version.range}",
+                            org.wso2.carbon.event.publisher.core; version="${carbon.analytics.common.version.range}",
+                            org.wso2.carbon.event.publisher.core.exception; version="${carbon.analytics.common.version.range}",
+                            org.wso2.carbon.event.publisher.core.config; version="${carbon.analytics.common.version.range}",
+                            org.wso2.carbon.event.stream.core.*; version="${carbon.analytics.common.version.range}",
```

Comment:
Please import specific packages instead of all.

Reply:
Fix https://github.com/wso2-extensions/identity-governance/pull/348